packages/reqresp/src/types.ts

```diff
@@ -129,7 +129,12 @@ export type HandlerTypeFromMessage<T> = T extends ProtocolGenerator ? ProtocolHa
 
 export type ContextBytesFactory =
   | {type: ContextBytesType.Empty}
-  | {type: ContextBytesType.ForkDigest; forkDigestContext: ForkDigestContext};
+  | {
+      type: ContextBytesType.ForkDigest;
+      forkDigestContext: ForkDigestContext;
+      fork: ForkName;
```

Comment:
same to the other comment, we only need to add an `epoch` here to simplify, no need `fork` and `blobSchedule`